### ISSUE TITLE
I want that the sign in page matches the theme of the whole website.

### ISSUE THREAD

**Issue by arunn0909** [arunn0909/BooksWorld#52]
Sign In and sign Up page is ready  now you just have to the changes to match it with the website theme.
current sign In page.
Note : Don't add any kind of background Images.
![signin](https://user-images.githubusercontent.com/114744225/193808816-0673d9c7-4cd8-4a1c-bc8a-26619f6a703a.png)

**Comment by shidoitsuka** [comment]
hi, you can assign me this
i'll start working on this

**Comment by shidoitsuka** [comment]
by the way, is it ok if i apply "minimalist web design" to this?

**Comment by arunn0909** [comment]
Assigning this issue to you @shidoitsuka  and sorry I don't have any idea regarding minimalist web design but you can look it by yourself I just want better output than the correct one.😊👍

**Comment by shidoitsuka** [comment]
i'm on it 😃

**Comment by shidoitsuka** [comment]
do you want the sign in page theme matches the index page :
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/34335880/193817460-653fc28d-bc8c-4316-b7a6-378e1e024002.png">
or the sign in page itself? because both currently have different color theme

**Comment by arunn0909** [comment]
Yes I want that signin page matches the theme of my index page or you can say website

**Comment by arunn0909** [comment]
@shidoitsuka bro are you working on it 🙄

**Comment by ayushmangarg2003** [comment]
Hey , Can I work on This Issue

**Comment by arunn0909** [comment]
Actually I have already assigned this task to someone. @ayushmangarg2003
